operator/cmd/mesh/testdata/manifest-generate/output/gateways.yaml

```diff
@@ -1000,14 +1000,9 @@ spec:
         imagePullPolicy: IfNotPresent
         name: istio-proxy
         ports:
-        - containerPort: 15020
-        - containerPort: 80
-        - containerPort: 443
-        - containerPort: 15029
-        - containerPort: 15030
-        - containerPort: 15031
-        - containerPort: 15032
-        - containerPort: 15443
+        - containerPort: 15011
```

Comment:
this block isn't right. I am not sure what the impact is, and I recognize this is a cherry-pick, however, 1006,1008 duplicate 1003, 1004.

Reply:
this is test data

Reply:
WFM - although I am surprised the tests would pass with invalid gateway config (unless that is what the test is testing..).

Reply:
it's just golden output, not input to e2e test.